data/strings_str.dat

```diff
@@ -50,3 +50,9 @@ money_per_day = Money per day.
 location_modifiers = Bases at this location are %(modifiers)s.
 cpu_bonus = more efficient
 cpu_penalty = less efficient
+
+stat_cash_earned = Cash Earned
+stat_cpu_used = Cpu Used
+stat_tech_created = Tech Created
```

Comment:
I think `Techs researched` or `Techs completed` might be better. Alternative `Techs invented`

Reply:
Yes.

Reply:
+1 for "Techs researched."